en/developer_guide/logs.rst

```diff
@@ -0,0 +1,92 @@
+Logs
+====
+
+Our backoffice logging is based on `Monolog`_.
+Open Orchestra provides an interface to display all the user actions on the website.
+
+.. image:: ../../images/log_preview.png
+
+Configuration
+-------------
+
+Monolog uses channels to record the log. To link a channel and a record, you will have to define an handler.
+The configuration of the handler can be placed in the ``app/config/log.yml`` file.
```

Comment:
I think this is misleading. The configuration can be placed anywhere, as long as it is included somewhere. If we eventually decide to not provide the full filesystem of the symfony app this will not be true.
@nicolasThal what do you think ?